### ISSUE TITLE
'Variables' no longer set by default

### ISSUE THREAD

**Issue by FreeCutter** [Zolko-123/FreeCAD_Assembly4#264]
Working with v0.11.3 I have the following question:

Why is the "Variables" elemenent no longer set by default once a 'New Assembly' is introduced?

Currently I have the following issue in my workflow:

- A variable is is introduced in an Assembly and "Animate Assembly" gets available and could be used without any issue.
- This assembly is inserted to another Main-Assembly.
- In the main-assembly the "animate assembly" is deactivated.
- As soon as a variable is introduced to the main-assembly (or 'Add Variable' is clicked and without setting a 'Name' the OK is clicked) the function "animate assembly" is getting available as well as the variable from sub-assembly could be reached and used for the animation.

I would expect that "animate assembly" is active as soon as variables also in linked assemblies are in an assembly.
In older versions I remember that once a new assembly had been introduced the "Variables" element had been introduced by default.
In other words missing of "Variables" by default seems to be the bug?

Thanks for fixing in case it's actually a bug.

**Comment by Zolko-123** [comment]
re-added this in the new v0.11.4 release
